cmd/appliance/upgrade/prepare.go

```diff
@@ -530,7 +531,7 @@ func prepareRun(cmd *cobra.Command, args []string, opts *prepareUpgradeOptions)
 				"Content-Disposition": fmt.Sprintf("attachment; filename=%q", zipInfo.Name()),
 			}
 			if !opts.ciMode {
-				err = uploadWithProgress(ctx, pr, "uploading "+logServerZipName, zipInfo.Size(), headers)
+				err = uploadWithProgress(ctx, pr, "uploading "+localLogServerBundleName, zipInfo.Size(), headers)
```

Comment:
Small nitpick: It would be nice to log "uploading Appgate-SDP-6.2-Logserver.zip as logserver-6.2.zip" to make clear what's happening, at least if the name is different locally